particle-iot/wiring-preprocessor
# Add node@16 and npm@8 support and migrate to circleci

## Description
Makes the required changes so the library works with `node@16x` and `npm@8.x`.

## How to Test

1. Pull the branch: `chore/sc-104122/update-wiring-preprocessor-to-node-js-v16`
2. Update your local node version to `v16.x` and npm to version to `v8.x`. Use `nvm use 16` or equivalent.
3. Verify your updates: `node -v && npm -v`
4. Install dependencies: `npm i`
5. Run the tests `npm run test`

**Outcome**
All tests should pass. npm i should run without issues.

## Related / Discussions


- https://app.shortcut.com/particle/story/104122/update-wiring-preprocessor-to-node-js-v16-and-npm-v8

## Completeness
